hr_employee_calendar_planning/tests/test_hr_employee_calendar_planning.py

```diff
@@ -431,3 +431,21 @@ def test_employee_copy(self):
         self.assertNotEqual(
             self.employee.resource_calendar_id, employee2.resource_calendar_id
         )
+
+    def test_user_action_create_employee(self):
+        ctx = {
```

Comment:
Do this once in the `setUp` changing `cls.env`, and it's covered for the whole test.